uber/templates/preregistration/form.html

```diff
@@ -187,4 +187,6 @@
 
 {% include "preregistration/disclaimers.html" %}
 
+</div>
+
```

Comment:
This was something @timrxd noticed while making his previous PR.  On one of the pages there was a closing div that wasn't present on the other.

Our browsers seems to correct for this, but it's still better to have valid HTML.  (We should probably start running our rendered pages against a validator or something.)